src/components/SRNEntry.vue

```diff
@@ -273,64 +264,52 @@ export default {
-    - redirectTo - from URL
-    - redirectID - from URL
-    - tempUserIDList - which contains only the ID of the incorrect entry
-    - authType - from URL 
-    @params {String} - userID - ID of the incorrect entry field */
-    handleIncorrectEntry(userID){
-        var purposeParams = "incorrect-entry"
-        var tempUserIDList = [{userID: userID.toString(), valid: this.isCurrentUserValid}]
-        sendSQSMessage(
-            this.purpose,
-            purposeParams,
-            this.redirectTo,
-            this.redirectID,
-            tempUserIDList,
-            authType
-          );
+    /** This function handles all invalid/incorrect entries. An SQS message is sent to the queue in AWS.
+     * @param {String} userID - ID of the incorrect entry field
+     */
+    handleIncorrectEntry(userID) {
+      var purposeParams = "incorrect-entry";
+      var tempUserIDList = [
+        { userID: userID.toString(), valid: this.isCurrentUserValid },
+      ];
+      sendSQSMessage(
+        this.purpose,
+        purposeParams,
+        this.redirectTo,
+        this.redirectID,
+        tempUserIDList,
+        authType
+      );
     },
-    /* this method is called whenever + button is clicked. It authenticates the most recent typed ID. 
-    If the ID is invalid, then the login mesasge displays an error and the input field is cleared for the user to correct their entry.
-    @params {String} - userID - most recent ID  */
+
+    /** This method is called whenever + button is clicked. It authenticates the most recent typed ID.
+     * @param {String} userID - most recent ID
+     */
     async authenticateSRN(userID) {
       this.isLoading = true;
-      let userValidationResponse = await validateSRN(
-        userID,
-        this.validateCount,
-        this.isSingleEntryOnly,
-        this.redirectID,
-        this.isCurrentUserValid,
-        this.purpose,
-        this.purposeParams,
-        this.redirectTo
-      );
+      let userValidationResponse = await validateSRN(userID, this.validateCount);
       this.isCurrentUserValid = userValidationResponse.isCurrentUserValid;
       this.validateCount = userValidationResponse.validateCount;
       this.invalidLoginMessage = userValidationResponse.invalidLoginMessage;
       this.isLoading = false;
 
-      if(this.invalidLoginMessage != ""){
-        this.resetEntry(this.userIdListLength - 1)
+      if (this.invalidLoginMessage != "") {
+        this.resetEntry(this.userIdListLength - 1);
       }
     },
-    /* this method is called after the user clicks the submit button. Since all the ID's but the last are authenticated, this method calls authenticateSRN(). 
-    If the user is invalid and is being authenticated the first time, handleIncorrectEntry() is called. 
-    If the user is valid or is being authenticated the second time, the user is reidrected to their destination and an SQS message is sent. */
+
+    /** This method is called after the user clicks the submit button.
```

Comment:
This still doesn't tell what the method does, it only says when this method is invoked.